OneSignalSDK.Xamarin.Android/NotificationExtensionAttribute.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Diagnostics;
+using Android.App;
+using Android.Content;
+
+namespace Com.OneSignal.Android
+{
+    [Serializable]
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
+    public sealed class NotificationExtensionAttribute : Attribute, Java.Interop.IJniNameProviderAttribute
```

Comment:
this attribute allow us to customize the name of the generated ACW classes however we like.

see how it works here:
https://github.com/xamarin/java.interop/blob/main/src/Java.Interop.Tools.TypeNameMappings/Java.Interop.Tools.TypeNameMappings/JavaNativeTypeManager.cs#L301-L305